train_deep_controller.py

```diff
@@ -42,11 +47,19 @@ def main(
         external_force_2: str,
         method: str,
         batch_size: int,
+        controller_type: str,
+        # is_tuner: bool
 ):
+    controller_type = None if controller_type == 'None' else controller_type
```

Comment:
I would suggest creating separate file `config.py` and explicitly define list of all available controller types, i.e.
`CONTROLLER_TYPES=["None", "external", "external_derivatives", "internal"]`

Reply:
Just to get rid of string constants all over different files

Reply:
OK